bluebutton-data-model-rif-samples/src/main/resources/rif-static-samples/sample-a-beneficiaries.txt

```diff
@@ -1,2 +1,2 @@
-INSERT|567834|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066U|Doe|John|A|3456789|17-Mar-1981|3|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|C|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|BB|
\ No newline at end of file
+INSERT|567834|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066U|Doe|John|A|3456789|17-Mar-1981|3|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|S9956|S9956|S9956|S9956|S9956|S9956|S9956|S9956|S9956|S9956|0|0|123|123|123|123|123|123|123|123|123|123|||04|04|04|04|04|04|04|04|04|04|||H4567|H4567|H4567|H4567|H4567|H4567|H4567|H4567|H4567|H4567|H4567|0|555|555|555|555|555|555|555|555|555|555|555||002|002|002|002|002|002|002|002|002|002|002||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|
```

Comment:
Good enough for the moment, but @wryenmeek and friends will want us to push through more realistic values here in a follow-on PR.